apps/krew-website/postcss.config.cjs

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
```

Comment:
If you leave out this line, you will get the following error:

`[vite:css] path is not defined`

..might take you a while to figure out what's the deal 😅